lib/stripe/api_operations/request.rb

```diff
@@ -3,8 +3,19 @@ module APIOperations
     module Request
       module ClassMethods
         OPTS_KEYS_TO_PERSIST = Set[:api_key, :api_base, :client, :stripe_account, :stripe_version]
+        KNOWN_OPTS = Set[:api_key, :idempotency_key, :stripe_account]
+
+        def warn_on_opts_in_params(params)
```

Comment:
Can you move both this method and the constant above down to a new `private` section in the file? Lets also put a `private_constant :KNOWN_OPTS` under the `KNOWN_OPTS` declaration.